maskrcnn_benchmark/data/datasets/coco.py

```diff
@@ -8,10 +8,10 @@
 
 class COCODataset(torchvision.datasets.coco.CocoDetection):
     def __init__(
-        self, ann_file, root, remove_images_without_annotations, transforms=None
+        self, ann_file, root, name, remove_images_without_annotations, transforms=None
     ):
         super(COCODataset, self).__init__(root, ann_file)
-
+        self.name = name
```

Comment:
Let's maybe remove the `name` attribute in favour of the approach I mentioned just before?

What are your thoughts?

Reply:
I got it. This would be a better choice.